types/react-dates/index.d.ts

```diff
@@ -275,7 +275,63 @@ declare namespace ReactDates {
     type DayPickerRangeController = React.ClassicComponentClass<DayPickerRangeControllerShape>;
     var DayPickerRangeController: React.ClassicComponentClass<DayPickerRangeControllerShape>;
 
+    interface DayPickerSingleDateControllerShape {
+        date: momentPropTypes.momentObj | null,
+        onDateChange: (date: momentPropTypes.momentObj | null) => void,
+        focused: boolean,
+        onFocusChange: (arg: { focused: boolean }) => void,
+        onClose?: Function,
+        
+        keepOpenOnDateSelect?: boolean
+        isOutsideRange?: (day: any) => boolean,
+        isDayBlocked?: (day: any) => boolean,
+        isDayHighlighted?: (day: any) => boolean,
+        
+        // DayPicker props
+        renderMonth?: Function,
+        enableOutsideDays?: boolean,
+        numberOfMonths?: number,
+        orientation?: ScrollableOrientationShape,
+        withPortal?: boolean,
+        initialVisibleMonth?: Function,
+        firstDayOfWeek? : 0 | 1 | 2 | 3 | 4 | 5 | 6,
```

Comment:
Please move this to DayOfWeekShape
https://github.com/airbnb/react-dates/blob/master/src/components/DayPickerSingleDateController.jsx#L54
the rest is good, thank you

type DayOfWeekShape = 0 | 1 | 2 | 3 | 4 | 5 | 6;
https://github.com/airbnb/react-dates/blob/master/src/constants.js#L28